### PR TITLE
Everything is a Value.

### PR DESCRIPTION
**Pull request by pushplay**
I'm going to merge this immediately to update the page but this creates a place to leave comments.

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -317,83 +344,74 @@ Contacts are the people or businesses who buy products or services from you.  Th
...
-- **Managing Gift Cards:** Gift Cards are typically created based on Lightrail's default Gift Card Program, which sets sensible defaults for gift card behaviour. You can also use custom [Gift Card Programs](#reference/promotions/gift-card-programs) to set parameters for particular use cases. 
-- **Redeeming Gift Cards:** Gift Cards are redeemed by your customers in [checkout](#reference/transactions/transactions/checkout). 
+A Value (naturally) represents value in the Lightrail system.  This includes a gift card, a promotion, a coupon, store credit, etc.  Values can be created one at a time but are most often [issued](#reference/0/issues) from a Program.  Issuing Value from a Program makes it part of that Program which keeps different ways of using Value organized.
+ 
+Most Values will be accessed by a code, or attached to a [Contact](#reference/0/contacts).
 
-### Create a Gift Card [POST /values/giftCards]
+### Create a Promotion [POST /values/promotions]
```
Comment (Tim-Jordison) [inline apiary.apib]: Should this be: Create a Value?

Comment (pushplay) [reply apiary.apib]: Good catch.

File: apiary-sources/sections/authentication.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## Authentication
+Authentication is done via an authorization header. You can retrieve your Lightrail API key [here](https://www.lightrail.com).
```
[the remark this answers is not in the document]
Comment (pushplay) [reply apiary-sources/sections/authentication.md]: That's a good idea.  Let's hold off on that until the app is done going through a refresh.

File: apiary-sources/sections/paging.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    rel="prev",
+  <https://api/lightrail.com/v2/contacts?limit=100&next=eyJpZCI6ImplbiBpcyBhd2Vzb21lIHRvbyJ9>;
+    rel="next",
+  <https://api/lightrail.com/v2/contacts?limit=100&last=true>;
+    rel="last"
+```
+
+This example has links to the first page, the previous page, the next page, and the last page.  When already at the first page there will only be links to the next and last.  When already at the first page there will only be links to the first and prev.  If there are no results to page though or only 1 page there will not be any links.
+
+It's important to follow the links in the Link header and not construct the URLs on your own.
```
[the remark this answers is not in the document]
Comment (pushplay) [reply apiary-sources/sections/paging.md]: No `before` and `after` are explicitly implementation details that users should not rely on.  Is there a better way to make that clear?

File: apiary-sources/api-sections/programs.md [hunk 4]
```diff
@@ -1,9 +1,10 @@
 ## Programs [/programs]
 
-Programs define the behaviour of Value that you wish to issue. Programs are typically created through the web app, but an explanation is included here for context and insight into how `Values` function.
+Programs organize [Values](#reference/0/values) and Values can be [issued](#reference/0/issuing-value) from Programs. They are typically created through the web app.
```
Comment (Tim-Jordison) [inline apiary-sources/api-sections/programs.md]: ~They~ Programs are typically created through the web app.

File: apiary-sources/sections/paging.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (tjukes) [reply apiary-sources/sections/paging.md]: Maybe it's clear enough as is. I suppose we could mention that with this implementation of paging, there is no `offset` and if you want to get to a specific subset of your results, you should refine your filters and/or keep going through the `prev`/`next` links.

**Pull request merged**